ops/terraform/services/base/sensitive.tf

```diff
@@ -19,6 +19,7 @@ resource "aws_ssm_parameter" "common_sensitive" {
   name      = each.key
   overwrite = true
   type      = "SecureString"
+  tier      = "Intelligent-Tiering"
```

Comment:
I've added `Intelligent-Tiering` to each `resource` here as the keystore parameter is larger than the maximum supported size of 4 KB for the `Standard` tier of SSM parameters. Fortunately, `Intelligent-Tiering` will automatically create the keystore parameter as an `Advanced` tier parameter, which have a limit of 8 KB. 

`Advanced` SSM parameters have a cost of:

- $0.05 per month per `Advanced` parameter
- $0.05 per 10,000 API interactions

This means, this new parameter (per-environment) will cost about $2 per year to store and likely less than $1 per year to access, which I felt was acceptable. We can discuss if a different approach should be taken here, if necessary.